examples/gitlab/README.md

```diff
@@ -0,0 +1,48 @@
+## GitLab Push EventListener
+
+Creates an EventListener that listens for Gitlab webhook events.
+
+### Try it out locally:
+
+1. Create the service account:
+
+   ```shell script
+   kubectl apply -f examples/role-resources/triggerbinding-roles
+   kubectl apply -f examples/role-resources/
+   ```
+
+1. Create the Gitlab EventListener:
+
+   ```shell script
+   kubectl apply -f examples/gitlab/gitlab-push-listener.yaml
+   ```
+
+1. Port forward:
+
+   ```shell script
+   kubectl port-forward \
+    "$(kubectl get pod --selector=eventlistener=gitlab-listener -oname)" \
+     8080
+   ```
+
+   **Note**: Instead of port forwarding, you can set the `serviceType` to
```

Comment:
Someone who is unfamiliar with the Triggers project might not know what the `serviceType` is, so I think that we should add a link to the `serviceType` docs (https://github.com/tektoncd/triggers/blob/master/docs/eventlisteners.md#serviceType)

ex:
'...Instead of port forwarding, you can set the [`serviceType`](https://github.com/tektoncd/triggers/blob/master/docs/eventlisteners.md#serviceType) to...'